hooks/useProfile.ts

```diff
@@ -41,6 +44,12 @@ const useProfile = () => {
     }
   }, [data]);
 
+  useEffect(() => {
+    if (searchParams.get('editing') === 'true' && canEdit) {
+      setIsEditing(true);
+    }
+  }, [canEdit, searchParams, artistAddress]);
```

Comment:
are both the `useEffect` and the initial value of the `useState` doing the same thing?

Are both necessary?

Which one could be removed for `simpler` and more `clean` code?